cetmix_tower_yaml/tests/test_server_log.py

```
@@ -0,0 +1,126 @@
+            }
+        )
+
+    def test_yaml_export_contains_file_id(self):
+        """Exported YAML must include a file_id starting with the file's reference."""
+        data = yaml.safe_load(self.log.yaml_code)
+        # Ensure file_id is present
+        self.assertIn("file_id", data, "`file_id` is missing from YAML export")
+        # Allow for auto-appended suffixes, so only check prefix
+        self.assertTrue(
+            data["file_id"].startswith(self.file.reference),
+            f"`file_id` value '{data['file_id']}' should start with '{self.file.reference}'",
```

Ruff (0.11.9)

Line too long (93 > 88)

(E501)